src/Servers/IIS/IISIntegration/src/IISDefaults.cs

```diff
@@ -3,10 +3,22 @@
 
 namespace Microsoft.AspNetCore.Server.IISIntegration
 {
+    /// <summary>
+    /// Defaults to configure IIS Out-Of-Process with.
+    /// </summary>
     public class IISDefaults
     {
+        /// <summary>
+        /// Default authentication scheme, which is "Windows".
+        /// </summary>
         public const string AuthenticationScheme = "Windows";
+        /// <summary>
+        /// Default negotiate string, which is "Negotiate".
+        /// </summary>
         public const string Negotiate = "Negotiate";
+        /// <summary>
+        /// Default ntlm string, which is "NTLM".
```

Comment:
```suggestion
        /// Default NTLM string, which is "NTLM".
```

I don't know what this means, but ok. This const seems pretty meh